extensions/mutiny/runtime/src/main/java/io/quarkus/mutiny/runtime/MutinyInfrastructure.java

```diff
@@ -11,4 +15,23 @@
     public void configureMutinyInfrastructure(ExecutorService exec) {
         Infrastructure.setDefaultExecutor(exec);
     }
+
+    public void configureDroppedExceptionHandler() {
+        Infrastructure.setDroppedExceptionHandler(new Consumer<Throwable>() {
+            @Override
+            public void accept(Throwable throwable) {
+                Logger.getLogger(MutinyInfrastructure.class).error("Mutiny had to drop the following exception", throwable);
```

Comment:
Should we cache the logger?

Reply:
Can we add the drop site too?

Reply:
It should be infrequent enough, but I can try moving it back to the heap. In earlier iterations I had GraalVM compilation issues, but it was my bad understanding of the `ExecutionTime` scopes.